docs/source/guide/frontend.md

```diff
@@ -36,16 +36,17 @@ cd label-studio/scripts
 node get-lsf-build.js <branch-name-from-official-lsf-repo>
 ```
 
-### CDN 
-
-You can include `main.<hash>.css` and `main.<hash>.js` files from a CDN directly. Explore `https://unpkg.com/label-studio@<LS_version>/build/static/` (e.g. [0.7.3](https://unpkg.com/label-studio@0.7.3/build/static/) to find correct filenames of js/css. 
+### CDN width UNPKG.com
 
 ```xhtml
-<!-- Theme included stylesheets -->
-<link href="https://unpkg.com/label-studio@0.7.3/build/static/css/main.14acfaa5.css" rel="stylesheet">
+<!-- Include Label Studio stylesheet -->
+<link href="https://unpkg.com/label-studio@1.0.1/build/static/css/main.css" rel="stylesheet">
+
+<!-- Create the Label Studio container -->
+<div id="label-studio"></div>
 
-<!-- Main Label Studio library -->
-<script src="https://unpkg.com/label-studio@0.7.3/build/static/js/main.0249ea16.js"></script>
+<!-- Include the Label Studio library -->
+<script src="https://unpkg.com/label-studio@1.0.1/build/static/js/main.js"></script>
```

Comment:
it's better to stick to the `@latest`

Reply:
@keesschollaart81 what do you think about @latest ?